### ISSUE TITLE
The usage of `@/` and vite

### ISSUE THREAD

**Issue by liusida** [litegraph-tsx/litegraph.tsx#13]
Hi, guys. I am trying to integrate this into my new project. But I don't want to use Vite in my project. 

The problem is that all files in `src/nodes/` contains import statements like:

```
import { LiteGraph } from "@/litegraph.js";
```

And I don't know how to make it work without using Vite. Any hints?

**Comment by liusida** [comment]
maybe we also want to add the sourcemap to the built `litegraph.mjs`, so that we can see the source file in the browser.

![image](https://github.com/user-attachments/assets/f387f582-aabd-475a-91cc-2859b9b4a64a)

**Comment by daniel-lewis-ab** [comment]
Yeah, the demo site on github doesn't like the @'s either, but the solution
is probably to run the built files.  We're also about to start using
typescript more thoroughly and it doesn't run from source either.

**Comment by DrJKL** [comment]
Exactly, the source files need to be compiled/bundled, then you'll have functional dist files.

**Comment by DrJKL** [comment]
We could also add an action to use Vite to deploy the example: https://vitejs.dev/guide/static-deploy#github-pages
